cli/exchange/business.go

```diff
@@ -86,6 +86,17 @@ func BusinessAddPolicy(org string, credToUse string, policy string, jsonFilePath
 		cliutils.Fatal(cliutils.CLI_INPUT_ERROR, msgPrinter.Sprintf("Incorrect deployment policy format in file %s: %v", jsonFilePath, err))
 	}
 
+	cons := len(policyFile.Constraints)
+	if noConstraints {
+		if cons > 0 {
+			cliutils.Fatal(cliutils.CLI_INPUT_ERROR, msgPrinter.Sprintf("--no-constraints flag is specified but the given deployment policy contains %v constraints.", cons))
```

Comment:
I dont think we should give an error in this case.

Reply:
done